packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/multiphysics/DecapodesConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.multiphysics;
+
+import java.io.Serializable;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.models.dataservice.TerariumAsset;
+
+@Data
+@Accessors(chain = true)
+@TSModel
+public class DecapodesConfiguration extends TerariumAsset implements Serializable {
+	private ConfigurationHeader header;
```

Comment:
Because you are using `TerariumAsset` you do not need a `ConfigurationHeader`.
`TerariumAsset` provides an `id, name, description`, only the `parentContext` is missing, that's why I'm using the `ModelHeader` in ticket #2638
```java
private ModelHeader header;
private Object configuration;
@JsonProperty("parent_context")
private UUID parentContext;
```